doc/sphinx/source/developer_guide2/config_developper.inc

```diff
@@ -0,0 +1,100 @@
+.. _config_developper:
```

Comment:
Typo: `developper` --> `developer`

Reply:
@ledm https://www.merriam-webster.com/dictionary/developer - it's an English word, not Quebecois :rofl:

Reply:
Done!